modules/contentbox/models/system/CBHelper.cfc

```diff
@@ -197,8 +197,8 @@ component accessors="true" singleton threadSafe{
 	/**
 	* Get the site base SES URL
 	*/
-	function siteBaseURL(){
-		return replacenocase( getRequestContext().buildLink( '' ), "index.cfm", "" );

```

Comment:
Yes probably would be better to have event.isSSL() without the need to pass any parameter